kotsadm/web/src/components/AirgapUploadProgress.jsx

```diff
@@ -38,13 +38,24 @@ class AirgapUploadProgress extends React.Component {
       });
 
       const response = await res.json();
+
+      if (!response.ok) {
+        this.setState({
+          installStatus: "airgap_upload_error",
+          currentMessage: `Encountered an error while uploading: ${response.error}`
```

Comment:
same here, i think this should be `res` instead of `response`, and there probably isn't an `error` field on that. we should use the status code instead.

Reply:
there is an `error` function though https://developer.mozilla.org/en-US/docs/Web/API/Response/error